src/messages/rdf.js

```diff
@@ -0,0 +1,26 @@
+import { gettext as _, singleLineString } from 'utils';
+
+
+export var _tagNotAllowed = (tagName) => {
+  return {
+    code: `TAG_NOT_ALLOWED_${tagName.toUpperCase()}`,
+    message: _(`The <${tagName}> tag is not allowed`),
+    description: _(singleLineString`Your RDF file contains the <${tagName}> tag,
+      which is not allowed in an Add-on.`),
+  };
+};
+
+export var _tagObsolete = (tagName) => {
+  return {
+    code: `TAG_OBSOLETE_${tagName.toUpperCase()}`,
+    message: _(`The <${tagName}> tag is obsolete`),
+    description: _(singleLineString`Your RDF file contains the <${tagName}> tag,
+      which is obsolete.`),
+  };
+};
+
+export const TAG_NOT_ALLOWED_HIDDEN = _tagNotAllowed('hidden');
```

Comment:
I wonder could you iterate over the list to generate the constants that way you've only got to add to a list?